### PR TITLE
[EVNT-187] Add pr_check.sh to repository

### PR DESCRIPTION
**Pull request by jpramos123**

### CODE REVIEW

**Review by CodeHeeler — COMMENTED** [review]
cicd-common config file in gitlab was deprecated which is why when deploying we use --source=appsre instead, so I'm not sure if all of your CICD pieces are still accurate.  The deprecation notice redirects to here, which doesn't explain it in detail.

https://consoledot.pages.redhat.com/docs/dev/getting-started/ephemeral/deploying.html#_deploying_using_app_interface_configs_used_by_pr_check_sh

**Review by CodeHeeler — APPROVED** [review]

**Comment by jpramos123** [comment]
> cicd-common config file in gitlab was deprecated which is why when deploying we use --source=appsre instead, so I'm not sure if all of your CICD pieces are still accurate. The deprecation notice redirects to here, which doesn't explain it in detail.
> 
> https://consoledot.pages.redhat.com/docs/dev/getting-started/ephemeral/deploying.html#_deploying_using_app_interface_configs_used_by_pr_check_sh

In terms of ci-cd common, we are good, the script is going to deploy it using --source=appsre:
https://github.com/RedHatInsights/bonfire/blob/master/cicd/deploy_ephemeral_env.sh#L14-L16

Thanks

File: splunk-quarkus/pr_check.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+#IQE_PLUGINS="drift"
+#IQE_MARKER_EXPRESSION="smoke" # Need to check this
+#IQE_FILTER_EXPRESSION=""
+#IQE_CJI_TIMEOUT="30m"
+
+
+# Install bonfire repo/initialize
+CICD_URL=https://raw.githubusercontent.com/RedHatInsights/bonfire/master/cicd
+curl -s $CICD_URL/bootstrap.sh > .cicd_bootstrap.sh && source .cicd_bootstrap.sh
```
Comment (vkrizan) [inline splunk-quarkus/pr_check.sh:20]: This is bad in so many ways. :grimacing: (feel free to reach out for explanation)

**Comment by vkrizan** [comment]
Thanks @jpramos123 !

**Pull request merged**